src/everything.rs

```diff
@@ -1343,3 +1343,35 @@ impl Drop for Everything {
         MACRO_MAP.clear();
     }
 }
+
+#[cfg(feature = "internal_benches")]
+mod benchmark {
+    use super::*;
+    use divan::Bencher;
+
+    #[cfg(feature = "vic3")]
+    #[divan::bench(args = internal_benches::vic3::bench_mods())]
+    fn load_provinces_vic3(bencher: Bencher, (vanilla_dir, modpath): (&str, &PathBuf)) {
+        bencher
+            .with_inputs(|| {
+                Everything::new(None, Some(Path::new(vanilla_dir)), None, None, modpath, vec![])
+                    .unwrap()
+            })
+            .bench_local_refs(|everything| {
+                everything.fileset.handle(&mut everything.provinces_vic3, &everything.parser);
+            });
+    }
+
+    #[cfg(feature = "ck3")]
```

Comment:
This is just a style issue, but in every case in tiger the order for games is: ck3, vic3, imperator,, hoi4. I would like to keep it in that order here too.